gms/impl/src/main/java/com/linkedin/metadata/resources/entity/EntityResource.java

```diff
@@ -210,4 +202,17 @@
       return null;
     });
   }
+
+  @Action(name = "getNumEntities")
```

Comment:
I don't want to disrupt this PR too much - but what do you think about naming this getTotalEntityCount? getNumEntities is a bit too vague IMO

Reply:
done!